src/server_manager/ui_components/app-root.html

```diff
@@ -42,8 +43,20 @@
     <style>
       :host {
         width: 100%;
-        --primary-green: #00bfa5;
-        --light-gray: color: rgba(0,0,0,0.54);
+        --status-bar-width: 48px;
```

Comment:
Also, how did you choose this figure? If I set it to something obviously crazy, e.g. 200px, then I get a nasty jump when minimising/maximising.

Reply:
The width is defined in the mocks. I think the animation works at the current width, the effect you describe could be the interaction of the side bar and app drawer animations.